data/hai/hai reveal 1 intro.txt

```diff
@@ -967,6 +965,69 @@ ship "Geodesic Geocoris" "Geodesic Geocoris (Phylactery)"
 	description "This modified Geocoris has identical cargo capacity, but is the result of an alien technology being merged with a hybrid Human-Hai operating system. It has additional shield emitters in a geodesic pattern over its main cargo wings, and some kind of integrated system core which seems to have something of a life of its own. As such, some systems are built-in and unable to be removed. Nevertheless it is significantly more survivable than a traditional Geocoris and at least as useful."
 
 
+# The spike in interest leads to ladybugs appearing in the Hai Merchant (Human) fleets.	
+mission "Hai Leaks Response 4C"
+	landing
+	invisible
+	to offer
+		has "event: hai ladybug available"
+	on offer
+		event "humans adopt ladybugs" 20 30
```

Comment:
I agree with Warp that this event should just be called alongside the "hai ladybug available" event as to avoid needing an extra mission.

> This setup also lets them be added to fleets smoothly on any saves currently already having passed the point where "hai ladybug available" is applied.
> So I don't really see the problem?

The problem is that we should not be making changes that account for people who have already gone through this content because this is not release content. One of if not the main reason Hai Reveal was merged at the start of v0.9.17's development instead of being a part of v0.9.15 going out the door is exactly because it would allow us to break things without worrying about it. The reason we typically make extra missions like this is to maintain compatibility between version numbers. There is no such compatibility to keep now, so this is just churn and should therefore be removed/simplified.